scripts/post.lua

```diff
@@ -0,0 +1,22 @@
+local pipelineDepth = 15
+
+function init(args)
+   wrk.method = "POST"
+   wrk.body   = "foo=bar&baz=quux"
+   wrk.headers["Content-Type"] = "application/x-www-form-urlencoded"
+
+   if args[1] ~= nil then
+      pipelineDepth = args[1]
+   end
+
```

Comment:
It's soo noisy though since it's printed once per thread. Annoying when you want to copy results into github comments.